src/views/SessionList.vue

```diff
@@ -349,4 +349,18 @@ onMounted(() => {
 </script>
 
 <style scoped>
-</style>
\ No newline at end of file
+  ion-fab-button {
+    --background: var(--ion-color-step-150, #ffffff);
+    --background-hover: var(--ion-color-step-200, #f2f2f2);
+    --background-focused: var(--ion-color-step-250, #d9d9d9);
+    --color: var(--ion-color-primary, #3880ff);
```

Comment:
Just verifying that this is the desired look?

| main | FW-5625 |
|--------|--------|
| ![Screenshot 2024-01-10 at 1 14 32 PM](https://github.com/ionic-team/ionic-vue-conference-app/assets/13530427/517804b5-09f9-4b6d-a9fc-f55cb429e101) | ![Screenshot 2024-01-11 at 8 41 33 AM](https://github.com/ionic-team/ionic-vue-conference-app/assets/13530427/c3563854-92cf-4cb2-9840-470de06ca3cc) |

Reply:
Yup, that's correct. If you look at the Angular and React conference apps, it looks the same as those.